gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/source/DefaultAuthSourceService.java

```diff
@@ -94,6 +96,7 @@ public DefaultAuthSourceService(@Autowired JwtAuthSourceService jwtAuthSourceSer
     public Optional<AuthSource> getAuthSourceFromRequest(HttpServletRequest request) {
         AuthSourceService service = getService(AuthSourceType.JWT);
         Optional<AuthSource> authSource = service.getAuthSourceFromRequest(request);
+        String logMessage = "Authentication request towards the southbound service {} using the auth source {}";
```

Comment:
This could be outside the method on class level (static and final)

Reply:
I will fix it, thanks for noticing